Monika After Story/game/script-topics.rpy

```diff
@@ -4299,6 +4299,28 @@ label monika_write:
     m "Thanks for listening~"
     return
 
+init 5 python:
+      addEvent(Event(persistent.event_database,eventlabel="monika_writingtip4",category=['writing tips'],prompt="Writing tip #4",pool=True))
+  
+label monika_writingtip4:
+     m 3b "Here's Monika's Writing Tip of the Day!"
+     m "You know about writer's block, right?"
+     m 1e "I had it a lot, when I started writing."
+     m "Sometimes it was halfway through a draft, but more often before I even started."
```

Comment:
m 1l "Sometimes it was halfway through a draft, but more often before I even started."